packages/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -97,7 +116,7 @@ import Vue from "vue";
 import mixins from "vue-typed-mixins";
 import {
 	DUPLICATE_MODAL_KEY,
-	EnterpriseEditionFeature,
+	EnterpriseEditionFeature, FAKE_DOOR_FEATURES,
```

Comment:
nit: shouldn't this be in the next line?

Reply:
Fixed! Unused import :)